app/controllers/votes_controller.rb

```diff
@@ -41,9 +41,14 @@ def create
 
     AbilityQueue.add(post.user, "Vote Change on ##{post.id}")
 
+    post = Post.find(params[:post_id])
```

Comment:
You can use the reload method to reload an object from the database, which I think would be preferable for this case (still triggers the same query though).

```suggestion
    post.reload
```